packages/api/src/router/page.ts

```diff
@@ -129,16 +130,19 @@ export const pageRouter = createTRPCRouter({
         .all();
       const workspaceIds = result.map((workspace) => workspace.workspaceId);
 
-      return opts.ctx.db
-        .select()
-        .from(page)
-        .where(
-          and(
-            eq(page.workspaceId, opts.input.workspaceId),
-            inArray(page.workspaceId, workspaceIds),
-          ),
-        )
-        .all();
+      return opts.ctx.db.query.page.findMany({
+        where: and(
+          eq(page.workspaceId, opts.input.workspaceId),
+          inArray(page.workspaceId, workspaceIds),
+        ),
+        with: {
+          monitors: {
```

Comment:
Does it really work because the relation is on pagesToMonitors ?

Reply:
I renamed `monitorsToPages` to `monitors`. Just to use monitors all over the frontend. tbd.